sourcegraph/sourcegraph
# [fix] when adding new user external account, permission sync was not scheduled

We now schedule high priority permission sync when new external account is associated with the user.

## Test plan

Unit tests + manual test:
1. Add new user external account to existing user on Account Security page
2. Go to permissions
3. You should see a new high priority job with `Third-party login service added for the user` as a reason (which might be hidden in elipsis, see screenshot).
<img width="892" alt="Screenshot 2023-06-26 at 13 16 38" src="https://github.com/sourcegraph/sourcegraph/assets/9974711/fd5b83d0-c728-4627-8109-5b1293cbb63f">
